### PR TITLE
export: add page

### PR DESCRIPTION
**Pull request by Crease29**
- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/export.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+> Command to set environment variables.
+
+- Set a new environment variable:
+
+`export {{VARIABLE}}={{value}}`
+
+- Echo the value of an environment variable (don't forget the Dollar sign):
+
+`echo {{$VARIABLE}}`
```
Comment (sbrl) [inline pages/common/export.md]: This example doesn't appear to be for the `export` command?

File: pages/common/export.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# export
+
+> Command to set environment variables.
```
Comment (sbrl) [inline pages/common/export.md]: Can't the export command be used to 'export' a variable's value to sub-processes too?

File: pages/common/export.md (hunk 1 above)
Comment (Crease29) [reply pages/common/export.md]: You're right. Shall I remove that example?

File: pages/common/export.md (hunk 2 above)
Comment (Crease29) [reply pages/common/export.md]: It's open to interpretation, but yes. `export` defines variables in the **current** environment and therefore sub-processes can access them as well.
Shall I point out in the description that variables that are set with `export` are only present in the current session and will not be persisted?

File: pages/common/export.md (hunk 1 above)
Comment (agnivade) [reply pages/common/export.md]: Yes.

**Comment by agnivade** [comment]
`export` is a shell built-in actually and not a command. And also, does this work in Windows ? If not, please create duplicate copies inside the linux and osx folder.

**Comment by Crease29** [comment]
Thanks for your feedback @agnivade. Please review the changes I've made.

**Comment by sbrl** [comment]
Looks ok to me! Thanks, @Crease29 😺

**Pull request merged**